diff -Nru xml-commons-1.0.b2.orig/java/src/org/apache/env/Which.java xml-commons-1.0.b2/java/src/org/apache/env/Which.java
--- xml-commons-1.0.b2.orig/java/src/org/apache/env/Which.java
+++ xml-commons-1.0.b2/java/src/org/apache/env/Which.java
@@ -289,12 +289,12 @@
         if (null == hash)
             return;
 
-        Enumeration enum = hash.keys();
+        Enumeration keys = hash.keys();
         Vector v = new Vector();
 
-        while (enum.hasMoreElements())
+        while (keys.hasMoreElements())
         {
-            Object key = enum.nextElement();
+            Object key = keys.nextElement();
             String keyStr = key.toString();
             Object item = hash.get(key);
 
@@ -311,12 +311,12 @@
             }
         }
 
-        enum = v.elements();
+        Enumeration elems = v.elements();
 
-        while (enum.hasMoreElements())
+        while (elems.hasMoreElements())
         {
-            String n = (String) enum.nextElement();
-            Hashtable h = (Hashtable) enum.nextElement();
+            String n = (String) elems.nextElement();
+            Hashtable h = (Hashtable) elems.nextElement();
 
             reportHashtable(h, n, out);
         }
